cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py

```diff
@@ -33,4 +33,13 @@ def upgrade():
 
 
 def downgrade():
-    pass
+    connection = op.get_bind()
+    res = connection.execute(
+        flag_table.select().where(flag_table.c.name == "pip-dev-preview")
+    ).fetchone()
+    if res is None:
+        connection.execute(flag_table.insert().values(name="pip-dev-preview", active=True))
+    else:
+        connection.execute(
+            flag_table.update().where(flag_table.c.name == "pip-dev-preview").values(active=True)
+        )
```

Comment:
This change is in the wrong commit (should be in "Migration for disabling pip-dev-preview flag")

Reply:
In this matter, the commit order seem to be inverted as well: We should first remove the filters for the flag and then disable it in the DB.

Reply:
Good catch, Fixed it